dotCMS/src/main/java/com/dotcms/util/pagination/TemplateView.java

```diff
@@ -0,0 +1,410 @@
+package com.dotcms.util.pagination;
+
+import java.io.File;
+import java.util.Date;
+
+public class TemplateView {
+
+    private final String  identifier;
+    private final String  inode;
+    private final String  categoryId;
+    private final String  body;
+    private final String  selectedimage;
+    private final String  image;
+    private final boolean drawed;
+    private final boolean deleted;
+    private final boolean live;
+    private final boolean locked;
+    private final boolean working;
+    private final boolean isNew;
```

Comment:
Codacy found an issue: [Field isNew has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6042990)